tests/ui/attributes/invalid-doc-attr.rs

```diff
@@ -5,7 +5,7 @@
 //~^ ERROR can only be applied at the crate level
 //~| WARN is being phased out
 //~| HELP to apply to the crate, use an inner attribute
-//~| SUGGESTION #![doc(test(no_crate_inject))]
+//~| SUGGESTION !
```

Comment:
Since this now inserts a bang, rather than taking the code, inserting a bang and replacing the whole line, this annotation isn't completely clear (but hopefully makes sense in context?)

Reply:
I love tighter spans :smiley:  so much clearer what the user actually needs to add